src/node_process_methods.cc

```diff
@@ -89,6 +89,14 @@ static void Chdir(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+inline Local<ArrayBuffer> get_fields_array_buffer(
```

Comment:
It would be better to name it like `get_fields_array_buffer_from_first_argument`, or accept an argument specifying the index of the argument array, otherwise this new convention makes the code less easier to grok IMO.